### ISSUE TITLE
Improve Accessibility score on Login page by adding alt text to images

### ISSUE THREAD

**Issue by fionaaac** [CMU-313/Teedy#59]
The current metric score for Accessibility is 86 on the login page. One reason for this lower score is because of the missing [alt] attribute from the Teedy logo image tag. Since the logo is an informative element, alt text should be added to describe it.

**Comment by fionaaac** [comment]
I would like to work on this please!
